### PR TITLE
Add help text for 'circle wait -h'

### PR DESCRIPTION
**Pull request by kevinburke**

### CODE REVIEW

File: circle/main.go [hunk 1]
```diff
@@ -112,6 +112,15 @@ func doDownload(flags *flag.FlagSet) error {
 
 func main() {
 	waitflags := flag.NewFlagSet("wait", flag.ExitOnError)
+	waitflags.Usage = func() {
+		fmt.Fprintf(os.Stderr, `usage: wait
+
+Wait for builds to complete, then print a descriptive output on 
+success or failure.
+`)
+		waitflags.PrintDefaults()
```
Comment (ianmdawson) [inline circle/main.go:121]: Does PrintDefaults() print anything here?

Comment (kevinburke) [reply circle/main.go:121]: No but if you add flags to the command, this will cause them to show up automatically

Comment (ianmdawson) [reply circle/main.go:121]: Cool, thanks!

**Comment by ianmdawson** [comment]
Lgtm ✨

**Pull request merged**